core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -283,6 +341,42 @@ oppia.controller('ExplorationEditor', [
       });
     };
 
+    $scope.showWelcomeExplorationModal = function() {
+      $scope.welcomeModalIsOpening = true;
+      var startTutorial = $scope.startTutorial;
```

Comment:
Why not just refer to $scope.startTutorial() directly in line 363? Then you don't need this declaration.